TestsFoundation/TestsFoundation/API/MockAPI.swift

```diff
@@ -56,7 +56,10 @@ public class MockAPI: API {
         if let (data, response, error) = mocks[request] {
             var value: R.Response?
             if let data = data {
-                value = try! JSONDecoder().decode(R.Response.self, from: data)
+                value = try? requestable.decode(data)
```

Comment:
This is probably what we wanted all along. Do we have cases that rely on the generic JSONDecoder below, or could we just remove that?